README.md

```diff
@@ -59,6 +60,16 @@ procedures.
   very easy to clean up these account at the end of the day and keep the costs
   low.
 
+## Releases
+
+Master should be stable and usable, however we release a new version once enough
```

Comment:
Not sure about this.

Reply:
So... can you actually tell me what you would find better then?

Reply:
Not mention `master` at all. We are not testing it enough to declare it as stable.

Reply:
Better?